core/spec/models/spree/adjustment_spec.rb

```diff
@@ -13,6 +13,32 @@
 
   let(:adjustment) { Spree::Adjustment.create!(label: 'Adjustment', adjustable: order, order: order, amount: 5) }
 
+  describe 'scopes' do
```

Comment:
I think 
```
let(:complete_order) { Spree::Order.create! completed_at: Time.current }
let(:incomplete_order) { Spree::Order.create! completed_at: nil }
let(:adjustment_for_complete_order) { Spree::Adjustment.create!(label: 'Adjustment', adjustable: complete_order, order: complete_order, amount: 5) }
let(:adjustment_for_incomplete_order) { Spree::Adjustment.create!(label: 'Adjustment', adjustable: incomplete_order, order: incomplete_order, amount: 5) }
```
should be only once in ```describe 'scopes'``` block instead of being both in complete and incomplete order blocks.